src/xdp-utils.c

```diff
@@ -397,8 +397,11 @@ xdp_app_info_rewrite_commandline (XdpAppInfo *app_info,
           for (i = 1; commandline[i]; i++)
             g_ptr_array_add (args, maybe_quote (commandline[i], quote_escape));
         }
-      else
+      else if (quote_escape) {
         g_ptr_array_add (args, g_shell_quote (app_info->id));
+      } else {
+        g_ptr_array_add (args, g_strdup (app_info->id));
+      }
```

Comment:
Please follow the code style of the file. Curly braces are in their own lines

Reply:
fixed it.